src/duke/DeadlineException.java

```diff
@@ -0,0 +1,6 @@
+package duke;
+
+public class DeadlineException {
```

Comment:
Since you name this as an exception, do you intend to make this class an exception? So perhaps extend the `Exception` class? `EventException` and `ToDoException` also has the problem.